### DISCUSSION TITLE
Migrate repository to ETH GitLab

### DISCUSSION THREAD

**Discussion by peanutfun** [CLIMADA-project/climada_python#711] [Ideas]
ETH hosts its own instance of the open-source repository management service GitLab at https://gitlab.ethz.ch/. Moving to GitLab would have some advantages:
* Self-hosted repository
* Better issue management system
* Easy registration of own GitLab Runners for CI/CD

And also some possible disadvantages:
* Changing repository remotes might confuse users
* Lower Google visibility (?)

As far as I can see, there are no features of GitHub that are *not* available on GitLab.

**Comment by mmyrte** [comment]
There is no way to register as a non-ETH-member, blocking outside contributions.

**Reply by peanutfun** [reply]
Woops, I did not realize this. This is a huge disadvantage and definitely *not* what we want

**Comment by emanuel-schmid** [comment]
Another disadvantage I can see is: quite some effort.
🧐 

e.g. the gitlab runners, easy - sure! but when it comes to grant availability - definitely not for free!
